safetrace-app/AppDelegate.swift

```diff
@@ -88,3 +93,19 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
         environment.safeTrace.session.setAPNSToken(deviceToken)
     }
 }
+
+// MARK: - App Switcher Overlay
+extension AppDelegate {
+    private func showAppSwitcherOverlay() {
+        appSwitcherOverlayWindow = UIWindow()
+        appSwitcherOverlayWindow?.rootViewController = AppSwitcherOverlayViewController()
```

Comment:
Would this not work by just modally presenting `AppSwitcherOverlayViewController` on top of the top visible VC? Or does it have to be the UIWindow's rootViewController to work?

Reply:
that would also work, but it's cleaner to do it this way (rather than mess with the actual navigation stack)